src/pages/trade-types/cfds/index.tsx

```diff
@@ -13,13 +13,26 @@ const ThingsToKeep = Loadable(() => import('./_mind-when-trading'))
 const AvailableMarkets = Loadable(() => import('./_available-markets'))
 
 const CFD = () => {
+    const meta_attributes = {
+        og_title: localize('CFD trading | Trade types | Deriv'),
+        og_description: localize(
+            'Trade CFDs on multiple markets and enjoy high leverage and risk management tools, giving you a chance to amplify your potential profits and limit your losses.',
+        ),
+        keywords: [
+            localize('cfd'),
+            localize('cfd trading'),
+            localize('trade cfd/cfds'),
+            localize('cfds leverage'),
+        ],
```

Comment:
@nikolaiberestevich-deriv , can we double confirm with Swati about adding the keyword, as I remembered when I was adding the SEO for Deriv-X page, the keywords are only for SEO team reference.

Reply:
Ok, I'll confirm with her

Reply:
You are right, we don't need to use keywords. Thank you for advice, I'll make a commit to discard keywords.